README.md

```diff
@@ -11,7 +11,9 @@ See the [Chalk book](https://rust-lang.github.io/chalk/book/) for more informati
 
 ## FAQ
 
-**How does chalk relate to rustc?** The plan is to have rustc use the `chalk-engine` crate (in this repo), which defines chalk's solver. The rest of chalk can then be considered an elaborate unit testing harness. For more details, see [the Traits chapter of the rustc-dev-guide](https://rustc-dev-guide.rust-lang.org/traits/index.html).
+**How does chalk relate to rustc?** The plan is to have rustc use the `chalk-engine` crate (in this repo), which defines
```

Comment:
This text is actually dated. Maybe you can update it to refer to [this section of the chalk book](http://rust-lang.github.io/chalk/book/#chalk-works-by-converting-rust-goals-into-logical-inference-rules), which outlines the way we now envision rustc integration working?